### ISSUE TITLE
[Bazel CI] Error: no native function or rule 'java_proto_library'


### ISSUE THREAD

**Issue by sgowroji** [bazelbuild/rules_android#280]
CI: https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/4264#019323b6-06ed-4e6a-8bb1-93b327b1ef20

 Platform: Ubuntu

 Logs: 
```

Error: no native function or rule 'java_proto_library'
``` 
 Culprit: 

 Steps:
  ``` 
 git clone https://github.com/bazelbuild/rules_android
git reset 2db3d8fc28171b172c9bdafa1dffae9777f98c08  --hard
export USE_BAZEL_VERSION=471aca124968977ecf1920711811c37ce19c1aa7
bazel build  --enable_bzlmod=false --enable_workspace=true --java_runtime_version=17  //android/... //src/... -//src/java/com/example/sampleapp/... //test/... -//test/rules/... //toolchains/... //tools/... -//tools/android/... 
 ``` 
 CC Greenteam @fweikert

**Comment by ahumesky** [comment]
Thanks, this is being addressed as part of the 0.6.0 release: https://github.com/bazelbuild/rules_android/issues/278

**Comment by sgowroji** [comment]
Closing this issue as the CI is green and above issue is fixed.
